README.md

```diff
@@ -78,20 +78,21 @@ To compile the contracts:
 
 ---
 ### Run
-To deploy instances of the contracts for local development without prior knowledge to Hardhat, run the following command:
+To deploy instances of the contracts for local development without prior knowledge to Hardhat, first copy .env.example to .env and run the following command:
 ```shell
 ./go contracts:run
 ```
 
-This command starts up built-in Hardhat Network and migrates all contracts to the Hardhat Network instance.
+This command starts up built-in Hardhat Network and migrates all contracts to the Hardhat Network instance. The .env file has a hard-coded value for the BOSON_TOKEN address, which points to account #9 of the local hardhat network. This isn't an actual
+BOSON token address but just a valid address that will allow the deployment scripts to work. It's not necessary to deploy a BOSON token to run the uni tests locally (see Unit Tests section), as the unit tests deploy their own contract instances.
```

Comment:
s/uni test/unit tests/

Reply:
Changed. Will be included in next commit.